src/utils/index.ts

```diff
@@ -54,8 +54,8 @@ export const stacContext = (req: Request) => {
     id: isCloudStac ? "CLOUDSTAC" : "STAC",
     root,
     stacRoot: `${root}/${stac}`,
-    path: `${root}${path}`,
```

Comment:
STAC browser included a trailing `/` in the URLS, which broke many of the links URLs for each item and collection.